spec/rubocop/cli/options_spec.rb

```diff
@@ -1470,7 +1476,7 @@ def full_description_of_cop(cop)
       context 'when simple format is specified' do
         it 'outputs with simple format' do
           cli.run(['--format', 'simple', 'example.rb'])
-          expect($stdout.string)
+          expect($stdout.string) # rubocop:todo RSpec/PredicateMatcher
```

Comment:
[memo] We have temporarily disabled this offense because of incorrect autocorrect.

Reply:
It doesn't matter if code update uses autocorrection or not. It should be handcrafted updated to the expected code instead of the `# rubocop:todo` comment.

Reply:
Indeed it is. I updated this PR.